aten/src/ATen/native/cpu/MultinomialKernel.cpp

```diff
@@ -94,27 +90,12 @@ void multinomial_apply(Tensor& result, const Tensor& self, const int64_t n_sampl
 
```

Comment:
(commenting here bc gh won't let me comment above) what does "(will be incremented for lua compat by wrapper)" mean?

Reply:
I think the lua-comment can be safely removed. I guess it was related to the indicies, since first element in arrays in lua is at position 1...